### PR TITLE
Test comment

### PR DESCRIPTION
**Pull request by joskalenda**
#### comment counter testing on this branch

### CODE REVIEW

File: src/set_api.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 const BASE_URL = 'https://api.spaceflightnewsapi.net/v3/articles';
 const involvementApi = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
 const commentsUrl = '/apps/Tl04OrbFyu6fxKeYxZ5Q/comments';
-// const likesUrl = '/apps/Uinv3HFYXneji5y5pk1g/likes';
```
Comment (zieeco) [inline src/set_api.js:4]: - [ ] Optional
- Please remove unused code here and to all other places where you have it in your codes.  Unused commented code makes your code not clean
- Please also check for typos when making a commit 
- According to the best practice on javascript, please try and improve your naming convention of your variables

Comment (zieeco) [inline src/set_api.js:4]: - Please remove this also.

**Review by zieeco — APPROVED** [review]
### Hi @joskalenda 

You did a great job so far 👏🏼 👏🏼 
Your project is complete!
Congratulations! 🎉🎉 
Now there is nothing left to do but to merge it! 🏆 
 👍  Good job!


## STATUS: APPROVED :100: 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joskalenda** [comment]
Thanks for the approval

**Pull request merged**